stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/textinput/MessageInputView.kt

```diff
@@ -307,6 +310,14 @@ public class MessageInputView : ConstraintLayout {
         }
     }
 
+    private fun handleKeyStroke() {
+        if (messageText.isNotBlank()) {
```

Comment:
Minor question: are we sure we don't want to check for `isEmpty()` here instead? I'd count hitting "space" as having started typing too 😄